### DISCUSSION TITLE
SWAG docs are out of date

### DISCUSSION THREAD

**Discussion by kinland** [authelia/authelia#5985] [General]
The Authelia SWAG docs (https://www.authelia.com/integration/proxies/swag/) are out of date. It's difficult to figure out whether a change is still needed. I presume so, but the lines that the docs say to change are no longer present (there is no longer a `set $authelia_backend` line)

The top of the new `authelia-server.conf` file now says:
> \# Make sure that your authelia container is in the same user defined bridge network and is named authelia
\# Rename /config/nginx/proxy-confs/authelia.subdomain.conf.sample to /config/nginx/proxy-confs/authelia.subdomain.conf
\# Make sure that the authelia configuration.yml has 'path: "authelia"' defined

So does this mean that SWAG is now following Authelia's recommendation?

Looks like this changed back in April (the file has changed again since then): https://github.com/linuxserver/docker-swag/commit/9cdedad1c8bc1e8819f236074db3c25a413da8be

Furthermore, I believe some other instructions are out of date. Further down in the prerequisite steps, it says links to files that can be found at https://www.authelia.com/integration/proxies/nginx/#authelia-locationconf and https://www.authelia.com/integration/proxies/nginx/#authelia-authrequestconf ; the source page there shows different filenames and paths.

I followed those instructions and ended up getting these errors:
> swag authalia "auth_request" directive is duplicate

I believe the `swag\config\nginx\authelia-server.conf` file makes the instructions about `authelia/location.conf`/`authelia-location.conf` snippets at least partially obsolete; while these files have a lot of differences, both provide `/authelia` location.

I believe that `swag\config\nginx\authelia-location.conf` makes the instructions about `authelia/authrequest.conf` / `authelia-authrequest.conf` file obsolete. They aren't identical, but they're extremely close. (This is also where the "duplicate" directive was)

It's possible that there are some config changes still needed to make it properly serve from a subdomain but it's really difficult to piece together the proper steps to follow since both Authelia's docs and SWAG's guide is out of date (their guide is a blog post from 3 years ago)

**Comment by Trufax** [comment]
Iam facing the same issue. Is there somewhere an updated guide on this?

**Reply by james-d-elliott** [reply]
You can use the nginx example, or you can adapt the current ones for the changes to swag (I'd love some succinct feedback on what has to change). When I get the time I'll update them (I missed this original post whoops).
